src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.scss

```diff
@@ -5,6 +5,24 @@
     position: absolute;
   }
 
+  &__background-image {
+    width: 100%;
+  }
+
+  &__background-image-placeholder {
+    width: 100%;
+    height: 1000px;
+    border-radius: $border-radius--lg;
+    box-shadow: 0px 6px 12px rgba(0, 0, 0, 0.25);
```

Comment:
opaque-black

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1995/commits/f033a214d20db26a1e8d035a115ccd345272d54f